packages/priority-queue/src/test/node/spec/PriorityQueue.test.js

```diff
@@ -102,7 +95,7 @@ describe('PriorityQueue', () => {
         throw Error('not so happy');
       }
 
-      queue = new PriorityQueue();
```

Comment:
Weird testcase...it should not have worked at all because with the old default value `maxRetries` was `Infinity` but it was green :O Good example why I changed the default value.